package.json

```diff
@@ -29,6 +29,7 @@
     "path": "^0.12.7",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
+    "styled-components": "^5.1.0",
```

Comment:
did you wind up using this or could it maybe be pruned out to save bundle space?